mongodbatlas/resource_mongodbatlas_federated_database_instance.go

```diff
@@ -240,8 +241,9 @@ func schemaFederatedDatabaseInstanceStores() *schema.Schema {
 					Optional: true,
 				},
 				"cluster_id": {
-					Type:     schema.TypeString,
-					Optional: true,
+					Type:       schema.TypeString,
+					Optional:   true,
+					Deprecated: fmt.Sprintf(DeprecationByDateMessageParameter, "September 2024"),
```

Comment:
@Zuhairahmed Could you please confirm if the deprecation message is okay for you?

Reply:
LGTM